package.json

```diff
@@ -31,6 +31,7 @@
   "homepage": "http://material-ui.com/",
   "dependencies": {
     "inline-style-prefixer": "^0.6.6",
+    "lodash.flowright": "^3.2.1",
```

Comment:
@alitaheri That's good to know thanks. I'm wondering if babel is doing the same with https://github.com/kittens/lerna.